pom.xml

```diff
@@ -456,46 +479,114 @@
             </dependency>
             <dependency>
                 <groupId>org.wso2.appserver</groupId>
-                <artifactId>bar-app</artifactId>
+                <artifactId>jaxrs_basic</artifactId>
                 <version>${appserver.version}</version>
                 <type>war</type>
             </dependency>
             <dependency>
                 <groupId>org.wso2.appserver</groupId>
-                <artifactId>foo-app</artifactId>
+                <artifactId>simple-jax-rs-service</artifactId>
                 <version>${appserver.version}</version>
                 <type>war</type>
             </dependency>
             <dependency>
                 <groupId>org.wso2.appserver</groupId>
-                <artifactId>jaxrs_basic</artifactId>
+                <artifactId>jpa-contacts-database</artifactId>
+                <version>${appserver.version}</version>
+                <classifier>classes</classifier>
+                <scope>test</scope>
+            </dependency>
+            <dependency>
+                <groupId>org.wso2.appserver</groupId>
+                <artifactId>cdi-alternatives</artifactId>
```

Comment:
Since we are using JavaEE samples only for integration tests, I think we need to mark those dependencies as test dependencies